lib/stdlib/test/peer_SUITE.erl

```diff
@@ -51,7 +51,7 @@ alternative() ->
 
 groups() ->
     [
-        {dist, [parallel], [errors, dist, peer_down_crash, peer_down_continue, peer_down_boot,
+        {dist, [], [errors, dist, peer_down_crash, peer_down_continue, peer_down_boot,
```

Comment:
I wonder what exactly breaks when tests are running in parallel? Is it timetrap too tight (so multiple concurrent nodes are not able to connect in a timely manner)? Or is it some node name clash (which would be very surprising, as the while idea of `peer` is to avoid such cases).

One failure mode for `peer` may be caused by setting `ERL_FLAGS="-noshell`. As I discovered it makes the started peer node to hang indefinitely (because no code from `peer.erl` is ever involved, it is starting via undocumented `-user` parameter which simply does not take effect when ERL_FLAGS specify something else).

Reply:
I'll revert the change and we can see if the gh test run fails :) I don't recall exactly what the error was.

Reply:
So, I/we remembered/figured out again what the problem was. Due to the changes in #5610 the `dist_io_redirect` test case fails when run in parallel as a new warning is printed to the io captured there. @rickard-green will move just that test case out from the parallel run to fix the problem.